cumulusci/tasks/robotframework/tests/test_robotframework.py

```diff
@@ -252,6 +252,7 @@ def test_sources(self, mock_add_path, mock_robot_run):
         )
         self.assertNotIn("dummy1", sys.path)
         self.assertNotIn("dummy2", sys.path)
+        self.assertEquals(".", task.return_values["robot_outputdir"])
```

Comment:
It might be good to have another test to verify that if we set the outputdir option, it is reflected in the return values to make sure we aren't just always returning the default rather than actual.